src/jit/codegenxarch.cpp

```diff
@@ -1813,162 +1813,20 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                     assert(structDesc.eightByteCount == CLR_SYSTEMV_MAX_EIGHTBYTES_COUNT_TO_PASS_IN_REGISTERS);
 
                     regNumber retReg0 = REG_NA;
-                    emitAttr size0 = EA_UNKNOWN;
-                    unsigned offset0 = structDesc.eightByteOffsets[0];
+                    unsigned __int8 offset0 = 0;
```

Comment:
@LLITCHEV  regarding the issue that i have mentioned about Profiler leave callback.  Please add a TODO comment below the comment block that starts with "!! Note !!"

e.g. 
TODO - Unix - Amd64: If the method returns a struct containing gc-type fields in two registers, then we need to keep alive either RAX or RDX or both RAX/RDX depending on field type.